### ISSUE TITLE
[Request] Colored ratio column?

### ISSUE THREAD

**Issue by runaroundhome** [jfurrow/flood#408]
Any thoughts about adding a bit of color to the ratio column to indicate the amount seeded?

**Comment by noraj1337** [comment]
It's difficult to define a good scale, you can do something like:
+ <0.8 : red
+ >0.8 && 1< : yellow/orange
+ >1 && <2 : light green
+ >2 && <10 : green
+ >10 : dark green

but what about when you reach >100 or >1000 (light super-seeded files) ?

Or that can be a more graduated scale, that can use the same colours as the download completed.

**Comment by runaroundhome** [comment]
Yeah that works for me. Personally anything over 10 can stay dark green.
